euphoria-examples/src/main/java/cz/seznam/euphoria/examples/wordcount/AccessLogCount.java

```diff
@@ -0,0 +1,140 @@
+package cz.seznam.euphoria.examples.wordcount;
```

Comment:
no problem. thanks for the quick fix. will merge as soon as travis gives us green light. once again, many thanks!